client/blocks/blog-stickers/use-blog-stickers-query.js

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'react-query';
+import wp from 'calypso/lib/wp';
+
+export const useBlogStickersQuery = ( blogId, queryOptions = {} ) =>
+	useQuery( [ 'blog-stickers', blogId ], () => wp.req.get( `/sites/${ blogId }/blog-stickers` ), {
+		...queryOptions,
+		enabled: !! blogId,
+		staleTime: 1000 * 60 * 5, // 5 minutes
+		meta: {
+			persist: false,
```

Comment:
Any good reason not to persist them? Seems like they don't change that often, and if we re-query them on every page load, they'll mostly be fresh anyway?

Reply:
Added persistence back in [st blog stickers queries](https://github.com/Automattic/wp-calypso/pull/63454/commits/9b9f78643002c05c83f84e790adccb7275693b93) 👍🏻 I was wondering whether adding / removing a sticker should result in an immediate update. Right now, I'm using a `staleTime` of 5 minutes which means the query won't be refetched on every page load if there's cached data but only if the data is older. In practice it probably won't make a big difference.